loaddb.py

```diff
@@ -1224,22 +1217,22 @@ def run_timers(c, elapsed_time):
     timer.run(c, elapsed_time)
 
 def load_extensions():
```

Comment:
`imp` is deprecated, I hardcoded what it used to dynamically

Reply:
Looks good